megatron/neox_arguments/arguments.py

```diff
@@ -501,10 +505,10 @@ def get_deepspeed_main_args(self):
                 args_list.extend(
                     self.convert_key_value_to_command_line_arg("comment", comment)
                 )
-            # master_address = os.environ['SLURM_JOB_NODELIST'].split('\n')[0]
-            # args_list.extend(
-            #    self.convert_key_value_to_command_line_arg('master_addr', master_address)
-            # )
+        # master_address = os.environ['SLURM_JOB_NODELIST'].split('\n')[0]
```

Comment:
Are these changes formatting or intentional?

Reply:
The commented-out items were already commented out on the main branch; here, they're just moved down  (not sure why they're showing as deleted/re-added). The remaining changes here are intentional--it seemed like the best place to do the check and set force_multi.

Reply:
Ok. I just removed the indentation.